src/main/java/com/fabriccommunity/thehallow/block/HallowedInfestedBlock.java

```diff
@@ -0,0 +1,30 @@
+package com.fabriccommunity.thehallow.block;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.InfestedBlock;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.enchantment.Enchantments;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.mob.EndermiteEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.GameRules;
+import net.minecraft.world.World;
+
+public class HallowedInfestedBlock extends InfestedBlock {
+
+	public HallowedInfestedBlock(Block block, Settings settings) {
+		super(block, settings);
+	}
+	
+	@Override
+	public void onStacksDropped(BlockState state, World world, BlockPos pos, ItemStack stack) {
+		if (!world.isClient && world.getGameRules().getBoolean(GameRules.DO_TILE_DROPS) && EnchantmentHelper.getLevel(Enchantments.SILK_TOUCH, stack) == 0) {
+			EndermiteEntity endermite = (EndermiteEntity) EntityType.ENDERMITE.create(world);
+			endermite.setPositionAndAngles(pos.getX() + 0.5D, pos.getY(), pos.getZ() + 0.5D, 0.0F, 0.0F);
```

Comment:
If we have a way to obtain the orientation of the player who broke the block we can consider making the mite face the player

Reply:
Eh, I'd prefer to keep the functionality identical to the vanilla Infested blocks